File: lib/supabaseClient.js
import { createClient } from "@supabase/supabase-js";

const supabaseUrl = "https://yheyhjcdzljcpoputsvo.supabase.co";

const supabaseKey = "sb_publishable_XKyEzrlSkHF_EqNHpsnxrQ__suhLExR";

export const supabase = createClient(supabaseUrl, supabaseKey);

File: backend/backend/app/services/supabase_client.py
from supabase import Client, create_client

from app.config import Config

_client: Client | None = None


def get_supabase() -> Client:
    global _client

    if _client is None:
        Config.validate()
        _client = create_client(Config.SUPABASE_URL, Config.SUPABASE_KEY)

    return _client
